python/tvm/relax/frontend/onnx_frontend.py

```diff
@@ -554,6 +554,47 @@ def _impl_v13(cls, bb, inputs, attr):
         return bb.emit_te(topi.subtract, inputs[0], inputs[1])
 
 
+class Split(OnnxOpConverter):
+    """Converts an onnx Split node into an equivalent Relax expression."""
+
+    @classmethod
+    def _impl_v1(cls, bb, inputs, attr):
+        splits = attr.get("split", None)
+        if splits is not None and len(splits) > 1:
+            indices = []
+            index = 0
+            for i in splits[:-1]:
+                index += i
+                indices.append(index)
+        # When splits isnt specified divide evenly over axis.
+        else:
+            indices = attr["tvm_custom"]["num_outputs"]
```

Comment:
What's the 'tvm_custom' thing do?

Reply:
It's set by the frontend when reading the onnx graph. This is similar to what the relay onnx op does.

https://github.com/apache/tvm/blob/main/python/tvm/relay/frontend/onnx.py#L6752